src/app/main/main.c

```diff
@@ -294,6 +294,18 @@ process_signal(int sig)
   }
 }
 
+#ifdef _WIN32
+/** Activate SIGINT on reciving a control signal in console */
+static BOOL WINAPI
+process_win32_console_ctrl(DWORD ctrl_type) {
```

Comment:
Okay, I missed this one because of the type-name on its own line situation. The `{` at the end of the function line have to go on its own line to satisfy the code style checker.

Reply:
Sorry, I kept forgetting to pay attention to the style, I was a bit busy so I didn't pay much attention. However I have now made several stylistic changes and the code should match other code now, I even ran `make check` locally this time to make sure :smile:

I did not get any errors related to style, but there were some failed checks in unrelated areas. Hope everything goes smoothly this time!